### DISCUSSION TITLE
Kimler Ibax Alacak?

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#700] [Soru Cevap - Türkçe] [answered]
Testnet i Yapan herkes İbax alabilicekmi yoksa  En çok referansı alan veya şanslı kişilermi alacak Cevaplarsanız sevirim :)

**Comment by Yusufknk** [comment, marked as answer]
Katılan herkes topladığı puanlara göre coin alacaklar.

**Reply by Skips1** [reply]
Yani  Herkez az da olsa alacak diyorsunuz yani Açıklamarda Bir kısmı En çok refrası getiren 2 k kişi ve geri kalanlar ise Çekiliş tarzı alınacak diyordu ? Hangisi doğru

**Reply by Yusufknk** [reply]
Airdrop 2 şekilde olacak A(referans ödülleri) ve B(retweet görevleri) ikiside ayrı hesaplanacak ve herkes coin alacak.

**Reply by frkselek** [reply]
jackpot A ve jacpot B olarak iki farklı ödül havuzu var. Rules kısmında detayları var sitede

**Reply by hes91** [reply]
300.000 IBXC katılımcılar arası bölüştürülecek.

**Reply by Tugbaturkekul** [reply]
teşekkürler bilgilendirme için

**Reply by ghost** [reply]
Kazanan gene yutubırlar varya

**Comment by Muhammedali26** [comment]
Katılan herkes alacak ancak kazandığı puanlara göre alacak. Ayrıca sanırım ilk 100 kişiye büyük miktarda verilecek ve yine bütün katılımcılar arasında yapılacak çekilişle 7 kişiye yine büyük miktarlarda dağıtım olacak

**Reply by ghost** [reply]
Bu yutubırlar götürecek gene :(

**Comment by Efsane4167** [comment]
Bence dagitim adil olmayacak.hintliler fake hesaplarla girdiler
